### PR TITLE
Fiorina Nightmare Insert

### PR DESCRIPTION
**Pull request by BadAtThisGame302**
# About the pull request

The Fiorina Prison has seen more and more unrest the last few years, quelling it down with the dwindling manpower. One such instance, the CMB picked up a beacon that was sent from the Prison. Thinking it was another riot, they asked for some Riot Control officers and sent them with a CMB Deputy to oversee the pacification of the station.

With no greetings at the LZ, they headed to the Armory to speak to someone in command. What they did not account for was that.. this was no riot, not anymore at least. The Security Hub is littered with both prisoner and Security forces corpses. Some sort of green blood and heavy fortifications. A team member snatched in the darkness and his screams silencing after a couple of seconds prove that... there's nobody left. 

You must survive until the CMB or some other force finds you.. and saves you before it's too late.

# Explain why it's good for the game

Woohooo nightmare insert baby yeayh

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

image

</details>

# Changelog

:cl:
mapadd: added a new Fiorina nightmare insert with CMB and UA Officers responding to the distress signal.
/:cl:

### CODE REVIEW

File: maps/map_files/FOP_v3_Sciannex/standalone/riot_in_progress.dmm
Comment (Nanu308) [inline maps/map_files/FOP_v3_Sciannex/standalone/riot_in_progress.dmm:1]: See Discord comment

File: code/game/objects/effects/landmarks/survivor_spawner.dm
Comment (Nanu308) [inline code/game/objects/effects/landmarks/survivor_spawner.dm:1]: Run the fluff text by the lore team if you haven't already, but from a quick read seems fine. 👍

**Review by Nanu308 — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/landmarks/survivor_spawner.dm
Comment (BadAtThisGame302) [reply code/game/objects/effects/landmarks/survivor_spawner.dm:1]: Kaga fluffed it up, should be good.

File: maps/map_files/FOP_v3_Sciannex/standalone/riot_in_progress.dmm
Comment (BadAtThisGame302) [reply maps/map_files/FOP_v3_Sciannex/standalone/riot_in_progress.dmm:1]: Removed hull walls.

**Review by Nanu308 — APPROVED** [review]
mapping lgtm

**Review by harryob — APPROVED** [review]

**Comment by Drulikar** [comment]
CI requires fixes from #5891

**Pull request merged**